soh/soh/Enhancements/randomizer/3drando/fill.cpp

```diff
@@ -488,14 +488,16 @@ static void GeneratePlaythrough() {
 RandomizerArea LookForExternalArea(Area* curRegion, std::vector<RandomizerRegion> alreadyChecked){//RANDOTODO curREGION
   for (auto& entrance : curRegion->entrances) {
```

Comment:
`auto& entrance` is not modified in this cycle, so you can add 'const' to it, making it 
``const auto& entrance``

Reply:
Also, please consider replacing this whole for-cycle with a known algo from C++ std algorithms library.
We are not transforming anything, just making a search - this cycle could be `std::find_if` with a predicate.